nupic/frameworks/opf/clamodel.py

```diff
@@ -235,19 +235,18 @@ def getParameter(self, paramName):
         (paramName))
 
 
-  def resetSequenceStates(self):
+  def resetSequenceStates(self, resetAnomaly=False):
```

Comment:
OPF `anomalyParams['claBunrInPeriod'] >= 0` resets anomaly, None to ignore (default)